### PR TITLE
Add mutual auth for RPC client / server.

### PR DESCRIPTION
**Pull request by Alan-Jowett**
RPC client and server should authenticate each other, to ensure:

Only admin client makes a call to ebpfsvc.
Client is talking only to ebpfsvc.

Resolves: #249 

Signed-off-by: Alan Jowett <alanjo@microsoft.com>

### CODE REVIEW

File: ebpfapi/rpc_client.cpp [hunk 1]
```diff
@@ -7,11 +7,12 @@
 #include <stdlib.h>
 #include <windows.h>
 #include "ebpf_api.h"
+#include "ebpf_platform.h"
 #include "rpc_interface_c.c"
+#include <sddl.h>
```
Comment (dthaler) [inline ebpfapi/rpc_client.cpp:12]: Move up to line 6 if possible

File: ebpfsvc/rpc_util.cpp [hunk 2]
```diff
@@ -4,11 +4,13 @@
 #include <malloc.h>
 #include "rpc_interface_s.c"
 #include "svc_common.h"
+#include <sddl.h>
```
Comment (dthaler) [inline ebpfsvc/rpc_util.cpp:7]: Move up before line 5

File: ebpfapi/rpc_client.cpp [hunk 3]
```diff
@@ -4,12 +4,12 @@
 #include <ctype.h>
 #include <iostream>
 #include <stdio.h>
+#include <sddl.h>
```
Comment (dthaler) [inline ebpfapi/rpc_client.cpp:6]: move up a line, in alphabetical order

File: ebpfsvc/rpc_util.cpp (hunk 2 above)
Comment (dthaler) [reply ebpfsvc/rpc_util.cpp:8]: Thanks for adding the comment.  I'd suggest between lines 4 and 5 explicitly including <windows.h> and then <sddl.h> (with the comment in line 7), and moving lines 5-6 down after the <> includes.

**Review by dthaler — APPROVED** [review]

**Review by shankarseal — APPROVED** [review]

**Pull request merged**